tagDataLocal.js

```diff
@@ -0,0 +1,268 @@
+                "desc"  : "Instead of Moving this turn, Unit may recover 1/3 round-down Armor points. All Attacks by this Unit this turn are at -4 ATK Dice"
+            },
+            {
+                "name"  : "Sharpshooter",
+                "id"    : 29,
+                "func"  : "tag_Sharpshooter",
+                "desc"  : "Unit does not suffer Stress penalty for targeting non-closest Enemy Unit"
+            },
+            {
+                "name"  : "Stable Fire Platform",
+                "id"    : 30,
+                "func"  : "tag_StableFirePlatform",
+                "desc"  : "Unit gains an additional +1 ATK when declaring Stationary during the Movement Phase"
+            },
+            {
+                "name"  : "Stall Speed",
+                "id"    : 31,
+                "func"  : "tag_StallSpeed",
+                "desc"  : "Gives this model a minimum move value that they must use otherwise they are destroyed in the Movment Phase. This model cannot take Stable Firing Platform along with this.  IF model cannot complete this minimum move, it is destroyed in the Resolution Phase."
+            },
+            {
+                "name"  : "Terrifying",
+                "id"    : 32,
+                "func"  : "tag_Terrifying",
+                "desc"  : "When Unit has finished its move, all Enemy Units within Short Range immediately suffer 1 Stress Point"
+            },
+            {
+                "name"  : "Transform",
+                "id"    : 33,
+                "desc"  : "Create 2 Statlines for a this Unit.  All damage is kept between the two modes. Unit cannot Move OR Shoot when switching between the modes. When calculating Unit Point Costs for Structure - only count the highest Structure value of all the modes."
+            },
+            {
+                "name"  : "Transform Mobile",
+                "id"    : 34,
+                "desc"  : "Treat as Transform, calculate costs as normal for Transform. Then, add this cost. Allows Unit to MOVE even if it has switched modes this turn."
+            },
+            {
+                "name"  : "Transform Weapons",
+                "id"    : 35,
+                "desc"  : "Treat as Transform, calculate costs as normal for Transform. Then, add this cost. Allows Unit to SHOOT even if it has switched modes this turn."
+            },
+            {
+                "name"  : "Transport",
+                "id"    : 36,
+                "func"  : "tag_Transport",
+                "desc"  : "Unit may carry other Friendly Units. Total allowed number is based on the Size of the carried Units. Total Size of all carried units cannot exceed half of this Unit's Size value."
+            }
+        ]
+    };
+    return tagData;
+};
+
+/*
+    Also pack the equations from tagEquations.js into an array
+    of functions. Binding the array number to the tag JSON id number.
+    'fun'
+*/
+
+function tagDataLocalEquationList(){
+    return {
```

Comment:
This is probably not the easiest way to do it. I would probably put all of the functions into an object, like this:

```
functions = {
  // They need not be 1:1 with tags, just cost patterns
  armorPiercing: (mainUnitData) => {
    // Do the calculation
  }
  // Each other costing function
}
```

Then call them all without having to do another layer of mapping:

`functions[tag.func](mainUnitData)`